core/graph/src/main/java/uk/gov/gchq/gaffer/operation/export/graph/handler/ExportToOtherGraphHandler.java

```diff
@@ -0,0 +1,191 @@
+import uk.gov.gchq.gaffer.operation.export.graph.OtherGraphExporter;
+import uk.gov.gchq.gaffer.store.Context;
+import uk.gov.gchq.gaffer.store.Store;
+import uk.gov.gchq.gaffer.store.StoreProperties;
+import uk.gov.gchq.gaffer.store.library.GraphLibrary;
+import uk.gov.gchq.gaffer.store.operation.handler.export.ExportToHandler;
+import uk.gov.gchq.gaffer.store.schema.Schema;
+import uk.gov.gchq.koryphe.ValidationResult;
+import java.util.List;
+
+public class ExportToOtherGraphHandler extends ExportToHandler<ExportToOtherGraph, OtherGraphExporter> {
+    private static final String ID = "gaffer.store.id";
+
+    @Override
+    protected Class<OtherGraphExporter> getExporterClass() {
+        return OtherGraphExporter.class;
+    }
+
+    @Override
+    protected OtherGraphExporter createExporter(final ExportToOtherGraph export, final Context context, final Store store) {
+
+        return new OtherGraphExporter(
+                context.getUser(),
+                context.getJobId(),
+                createGraph(export, store));
+    }
+
+    protected Graph createGraph(final ExportToOtherGraph<?> export, final Store store) {
+
+        final String exportGraphId = export.getGraphId();
+        final Schema exportSchema = export.getSchema();
+        final StoreProperties exportStoreProperties = export.getStoreProperties();
+        final List<String> exportParentSchemaIds = export.getParentSchemaIds();
+        final String exportParentStorePropertiesId = export.getParentStorePropertiesId();
+
+        final GraphLibrary graphLibrary = store.getGraphLibrary();
+        final StoreProperties storeStoreProperties = store.getProperties();
+        final Schema storeSchema = store.getSchema();
+
+        ValidationResult validationResult = validate(exportGraphId, exportParentSchemaIds, exportParentStorePropertiesId,
+                graphLibrary, exportSchema, exportStoreProperties, store);
+        if (!validationResult.isValid()) {
+            throw new IllegalArgumentException(validationResult.getErrorString());
+        }
+
+        StoreProperties storeProperties;
+        Schema schema;
+
+        // No store graph library so we create a new Graph
+        if (null == graphLibrary) {
+            schema = null != exportSchema ? exportSchema : storeSchema;
+            final StoreProperties properties = null != exportStoreProperties ? exportStoreProperties : storeStoreProperties;
+            return new Graph.Builder()
+                    .graphId(exportGraphId)
+                    .addSchema(schema)
+                    .storeProperties(properties)
+                    .build();
+        }
+
+        // If the graphId exists in the graphLibrary then just use it
+        if (graphLibrary.exists(exportGraphId)) {
+            return new Graph.Builder()
+                    .graphId(exportGraphId)
+                    .library(graphLibrary)
+                    .build();
+        } else {
```

Comment:
Quite a large method, Could be split up into smaller methods with meaningful name. Especially
this large else block.